tensorflow/lite/kernels/internal/optimized/optimized_ops.h

```diff
@@ -7730,6 +7728,219 @@ inline void BroadcastPReluDispatch(
                           PReluElementWise, PReluScalarBroadcast);
 }
 
+// Returns the index with minimum value within `input_data`.
+// If there is a tie, returns the smaller index.
+template <typename T>
+inline int ArgMinVector(const T* input_data, int n) {
+  T min_value = input_data[0];
+  int min_index = 0;
+  for (int i = 1; i < n; ++i) {
+    const T curr_value = input_data[i];
+    if (curr_value < min_value) {
+      min_value = curr_value;
+      min_index = i;
+    }
+  }
+  return min_index;
+}
+
+// Returns the index with maximum value within `input_data`.
+// If there is a tie, returns the smaller index.
+template <typename T>
+inline int ArgMaxVector(const T* input_data, int n) {
+  T max_value = input_data[0];
+  int max_index = 0;
+  for (int i = 1; i < n; ++i) {
+    const T curr_value = input_data[i];
+    if (curr_value > max_value) {
+      max_value = curr_value;
+      max_index = i;
+    }
+  }
+  return max_index;
+}
+
+template <>
+inline int ArgMinVector(const float* input_data, int size) {
```

Comment:
Choose the only one keyword from among the `n` or `size` keywords for the template functions consistently.

Reply:
My recommendation is using the size keyword.